## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 477291`
Commit: `4f8c7f8ca95376a0d206ef38edfc74498af86885`

---
### `x86_64-linux`
<details>
  <summary>:x: 59 packages failed to build:</summary>
  <ul>
    <li>home-assistant</li>
    <li>home-assistant-component-tests.alexa</li>
    <li>home-assistant-component-tests.august</li>
    <li>home-assistant-component-tests.automation</li>
    <li>home-assistant-component-tests.cast</li>
    <li>home-assistant-component-tests.cloud</li>
    <li>home-assistant-component-tests.default_config</li>
    <li>home-assistant-component-tests.google_assistant</li>
    <li>home-assistant-component-tests.http</li>
    <li>home-assistant-component-tests.loqed</li>
    <li>home-assistant-component-tests.matter</li>
    <li>home-assistant-component-tests.miele</li>
    <li>home-assistant-component-tests.mobile_app</li>
    <li>home-assistant-component-tests.netatmo</li>
    <li>home-assistant-component-tests.onedrive</li>
    <li>home-assistant-component-tests.overseerr</li>
    <li>home-assistant-component-tests.owntracks</li>
    <li>home-assistant-component-tests.plaato</li>
    <li>home-assistant-component-tests.rachio</li>
    <li>home-assistant-component-tests.recovery_mode</li>
    <li>home-assistant-component-tests.script</li>
    <li>home-assistant-component-tests.toon</li>
    <li>home-assistant-component-tests.withings</li>
    <li>home-assistant-component-tests.yale</li>
    <li>home-assistant-component-tests.yolink</li>
    <li>home-assistant-custom-components.dreo</li>
    <li>home-assistant-custom-components.elegoo_printer</li>
    <li>home-assistant-custom-components.frigate</li>
    <li>home-assistant-custom-components.hass_web_proxy</li>
    <li>home-assistant-custom-components.homematicip_local</li>
    <li>home-assistant-custom-components.midea_ac</li>
    <li>home-assistant-custom-components.moonraker</li>
    <li>home-assistant-custom-components.mypyllant</li>
    <li>home-assistant-custom-components.octopus_energy</li>
    <li>home-assistant-custom-components.oref_alert</li>
    <li>home-assistant-custom-components.pirate-weather</li>
    <li>home-assistant-custom-components.powercalc</li>
    <li>home-assistant-custom-components.sleep_as_android_mqtt</li>
    <li>home-assistant-custom-components.thewatchman</li>
    <li>home-assistant-custom-components.yandex-station</li>
    <li>home-assistant.dist</li>
    <li>python-matter-server</li>
    <li>python-matter-server.dist</li>
    <li>python313Packages.home-assistant-chip-clusters</li>
    <li>python313Packages.home-assistant-chip-clusters.dist</li>
    <li>python313Packages.home-assistant-chip-core</li>
    <li>python313Packages.home-assistant-chip-core.dist</li>
    <li>python313Packages.home-assistant-chip-wheels</li>
    <li>python313Packages.homeassistant-stubs</li>
    <li>python313Packages.homeassistant-stubs.dist</li>
    <li>python313Packages.python-matter-server</li>
    <li>python313Packages.python-matter-server.dist</li>
    <li>python314Packages.home-assistant-chip-clusters</li>
    <li>python314Packages.home-assistant-chip-clusters.dist</li>
    <li>python314Packages.home-assistant-chip-core</li>
    <li>python314Packages.home-assistant-chip-core.dist</li>
    <li>python314Packages.home-assistant-chip-wheels</li>
    <li>python314Packages.python-matter-server</li>
    <li>python314Packages.python-matter-server.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 14 packages built:</summary>
  <ul>
    <li>python313Packages.databackend</li>
    <li>python313Packages.databackend.dist</li>
    <li>python313Packages.pins</li>
    <li>python313Packages.pins.dist</li>
    <li>python313Packages.pip-tools</li>
    <li>python313Packages.pip-tools.dist</li>
    <li>python313Packages.sphinx-markdown-builder</li>
    <li>python313Packages.sphinx-markdown-builder.dist</li>
    <li>python314Packages.databackend</li>
    <li>python314Packages.databackend.dist</li>
    <li>python314Packages.pip-tools</li>
    <li>python314Packages.pip-tools.dist</li>
    <li>python314Packages.sphinx-markdown-builder</li>
    <li>python314Packages.sphinx-markdown-builder.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python313Packages.home-assistant-chip-wheels</summary>
<pre>FAILED: [code=1] python/gen/matter_build_venv._install_base_3p_deps/pip_install_log.txt python/gen/matter_build_venv._install_base_3p_deps.pw_pystamp 
python3 ../../third_party/pigweed/repo/pw_build/py/pw_build/python_runner.py --gn-root ../../ --current-path ../.. --default-toolchain=//build/toolchain/custom:custom --current-toolchain=//third_party/pigweed/repo/pw_build/python_toolchain:python --touch python/gen/matter_build_venv._install_base_3p_deps.pw_pystamp --capture-output --module pip --python-virtualenv-config python/gen/matter_build_venv/venv_metadata.json --python-dep-list-files python/gen/matter_build_venv._install_base_3p_deps_metadata_path_list.txt -- install --requirement ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt --log python/gen/matter_build_venv._install_base_3p_deps/pip_install_log.txt
Looking in links: file:///nix/store/9am0i27ipv3zba1wb58kvm4s3jshn5nd-python3.13-aiohttp-3.13.2-dist, file:///nix/store/lwxicdz7v9sjr9nn2kc2yjwrggvlx9w4-python3.13-alive-progress-3.1.5-dist, file:///nix/store/z44sw3wqdk5w3kkzv9fahcrah2bp8h6i-python3.13-colorama-0.4.6-dist, file:///nix/store/482jjs02jvj7zm57d3ca3ddlmr51i1ps-python3.13-coloredlogs-15.0.1-dist, file:///nix/store/nvzyv3fvd6p33dxmhar1lynd90fvwyvx-python3.13-click-8.3.1-dist, file:///nix/store/n5z5hz1cqybk8py0rhdz408kxn1pddy5-python3.13-cryptography-46.0.3-dist, file:///nix/store/md7r2l3rwn4mr1zx936r3qvf4iw5b09w-python3.13-debugpy-1.8.19-dist, file:///nix/store/kf9bkcfk4pgzp7zk3vmki9bfpcqk7sh0-python3.13-diskcache-5.6.3-dist, file:///nix/store/p4cw1grlmh9dla1ag0wvavfz0x8ng24n-python3.13-googleapis-common-protos-3.31.3-dist, file:///nix/store/7y3v5gym9bmkg20brbzrimyd9yl3kzal-python3.13-ipython-9.7.0-dist, file:///nix/store/lzzi0r42dxxxp10w45687jr754gjb9kc-python3.13-jinja2-3.1.6-dist, file:///nix/store/qwd6099rz6422i5b6i4d5gxnr8g7q0l2-python3.13-json5-0.12.1-dist, file:///nix/store/mdj1y2n4h5h43pvpkik76549r7y5kcv9-python3.13-jsonschema-4.25.0-dist, file:///nix/store/hjbxv619hls9flwym1zbdvxrv2wxa0qr-python3.13-lark-1.2.2-dist, file:///nix/store/rd37zyxfh275xks3xs4na867v6xpycv5-python3.13-mobly-1.13-dist, file:///nix/store/c6pf23jrw3fmgfhy033sknj9xbq17927-python3.13-mypy-1.17.1-dist, file:///nix/store/pbk6ni2zwz88nl84n3kfm0chdi9bb8ia-python3.13-mypy-protobuf-3.6.0-dist, file:///nix/store/yxqidjfdisjjd8a01gsy3kcil4snl9zq-python3.13-packaging-25.0-dist, file:///nix/store/likbndbm5f02rhsznp1bnlbgf0bzy858-python3.13-parameterized-0.9.0-dist, file:///nix/store/x45444lh3apyrybbsb83sxy6kpr8cnvp-python3.13-pkgconfig-1.5.5-dist, file:///nix/store/bsai7apq9dcdr5qdwkmnh2hvh4m2071y-python3.13-prompt-toolkit-3.0.52-dist, file:///nix/store/k6pn9x8x7x36gx57y359lf67ama1k3zi-python3.13-protobuf-6.33.2-dist, file:///nix/store/svb84gyai1678g87bqsdxr21b59w5qfs-python3.13-psutil-7.1.3-dist, file:///nix/store/1gvrd50z05193wp856bp0dr1cvljxbrb-python3.13-ptpython-3.0.32-dist, file:///nix/store/1lyjxvxpwmww6pbzin2p6r1xri83qs8l-python3.13-pyfakefs-5.9.2-dist, file:///nix/store/3whmpizakqin7nmglikgrs25bm3g7d2d-python3.13-pyelftools-0.32-dist, file:///nix/store/79b6m3bilkks01lf84l63prv3y4f5fxn-python3.13-pygments-2.19.2-dist, file:///nix/store/62lh43c0bgs47l8wfpk33y55afrpp4al-python3.13-pykwalify-1.8.0-dist, file:///nix/store/azjzid3p5w0imlmfr3q6fjzqxn2pspcv-python3.13-pyperclip-1.11.0-dist, file:///nix/store/0v4b66jn77ixs66fnrp5npy6lr0jjy0y-python3.13-pyserial-3.5-dist, file:///nix/store/n6m22ndi0q3rq3ma1cwj0l4l6ghfxymx-python3.13-python-daemon-3.1.2-dist, file:///nix/store/i5bpjp0alqp214fgpy7pgbxp3xg1vjax-python3.13-python-path-0.1.3-dist, file:///nix/store/lfd25mwxnamlf4yvlsqpqrdff6wv0804-python3.13-pyyaml-6.0.3-dist, file:///nix/store/8fl2f2ksg0myc74mh4irmp902ys41kxz-python3.13-requests-2.32.5-dist, file:///nix/store/wn9lx4c94nanrrnpmq6pm0cacd9ibysw-python3.13-six-1.17.0-dist, file:///nix/store/4w75rylz1xnwnc4v1sism8kw8v280qji-python3.13-sphinx-8.2.3-dist, file:///nix/store/zzgkly1ymvw940p4bhc4nxnrnmis99yw-python3.13-sphinx-argparse-0.5.2-dist, file:///nix/store/985mmir7nkc88ml8rifdcn6bkvrg9ys6-python3.13-sphinx-design-0.6.1-dist, file:///nix/store/5f96h85si4679g3vk30bnww4d4gcnrv2-python3.13-tabulate-0.9.0-dist, file:///nix/store/n92dv9lqimjb30lngvfrczc82knjs4cy-python3.13-tornado-6.5.2-dist, file:///nix/store/nwy49plzg62fqcsymfjzag39xry3hi6j-python3.13-types-pyyaml-6.0.12.20250516-dist, file:///nix/store/3iwq898hs6mnng8x5hafdlxkwdkykns3-python3.13-types-requests-2.32.4.20250913-dist, file:///nix/store/gg906xzwf3491grmhxzpwhyiz22szk4n-python3.13-watchdog-6.0.0-dist, file:///nix/store/8d1mmv636y0ccf42fna03bfhi57whlsr-python3.13-websockets-15.0.1-dist, file:///nix/store/v0mv7srrf7w513v4cf8f5bgp8cc6rw7d-python3.13-aiohappyeyeballs-2.6.1-dist, file:///nix/store/jws116s2pwvla1gcym2hdgz0284bbmlh-python3.13-aiosignal-1.4.0-dist, file:///nix/store/dr3hla3rra9h59sckzg1lys2xgzkf676-python3.13-attrs-25.4.0-dist, file:///nix/store/flsx0vh9a4aynpx0frydra5pg1dj0xwd-python3.13-frozenlist-1.7.0-dist, file:///nix/store/b61n6s86w03f6kx6kgn3ggf0a6rq066l-python3.13-multidict-6.7.0-dist, file:///nix/store/d0iv4mzsng7zjhagix1cik92hpqc5src-python3.13-propcache-0.4.1-dist, file:///nix/store/9ilgi19lg8s3zhw8ipa97ir2655wca7r-python3.13-yarl-1.22.0-dist, file:///nix/store/fc3csl6rmah2m3i024q5v6lb4wb4n8w2-python3.13-aiodns-3.5.0-dist, file:///nix/store/xg21m8fly3mq0fycv93xk71qav8pj8mn-python3.13-brotli-1.2.0-dist, file:///nix/store/3119ymv0cqiva4qc8g4vnkhzjbv1aphd-python3.13-backports-zstd-1.1.0-dist, file:///nix/store/ja1ffajicwmyxhkklr2v7jd1gy6zq7d9-python3.13-about-time-4.2.2-dist, file:///nix/store/v25p44vg8342w8r6lr0z0wvxpp1d6q8n-python3.13-grapheme-0.6.0-dist, file:///nix/store/6spyfp37sycb64mjjg73cmvrfrg9n5gy-python3.13-humanfriendly-10.0-dist, file:///nix/store/cnwb4hr513izhc5s5adhaanh87qz6gn9-python3.13-cffi-2.0.0-dist, file:///nix/store/k7jz8aq9xichz5s93vgfsfvg21jhvllw-python3.13-decorator-5.2.1-dist, file:///nix/store/r5a2hn6ngsyi7c0zhqn4p03plvwkscra-python3.13-ipython-pygments-lexers-1.1.1-dist, file:///nix/store/qbnys07y3629b405c0hc7hrdlz81ygcv-python3.13-jedi-0.19.2-dist, file:///nix/store/x5vx4w8fbq52vsdxiaibymgk791nbpkz-python3.13-matplotlib-inline-0.1.7-dist, file:///nix/store/zpv80r8xpr7rmxspc11j73sg8h0zz91r-python3.13-pexpect-4.9.0-dist, file:///nix/store/amlg8c1sk7q17ib1vfnj8fpqa43csn25-python3.13-stack-data-0.6.3-dist, file:///nix/store/vkqaddik47mhlsgf9hidfyhsh79hhpqw-python3.13-traitlets-5.14.3-dist, file:///nix/store/dcd0qgfipbmqyi3jvy0vnnlvchngdk75-python3.13-markupsafe-3.0.3-dist, file:///nix/store/k80ylg0qrmcy5sky8j7fwqm1hmp1j2lq-python3.13-jsonpath-ng-1.7.0-dist, file:///nix/store/hdwqc0qiphq1q3d65rl7m25wizd5cp9i-python3.13-jsonschema-specifications-2025.4.1-dist, file:///nix/store/y15000z2z1fxqkwppihqg1j4h4zw0gal-python3.13-referencing-0.36.2-dist, file:///nix/store/2makjnq0zyq7cx20555li3zizwv77q2w-python3.13-rpds-py-0.28.0-dist, file:///nix/store/kpgsx6jpi0k4bb3891qrxj8idgnsws3f-python3.13-regex-2025.9.18-dist, file:///nix/store/2b4nxvjs5sv2qddzrzfyrn0lkw3fgvsx-python3.13-portpicker-1.6.0-dist, file:///nix/store/0kpnn7vjakgkgl7nm6zskk4vpzxvj0p7-python3.13-timeout-decorator-0.5.0-dist, file:///nix/store/2a5hzmb5g8y9yjr4janja1gi98f1y4xp-python3.13-typing-extensions-4.15.0-dist, file:///nix/store/jmywnxzs24d3pbf89q5z70n2nqbl2q6q-python3.13-mypy-extensions-1.1.0-dist, file:///nix/store/i1wjbwg8arfq8yn8h309nzs23dc3i3xm-python3.13-pathspec-0.12.1-dist, file:///nix/store/822swgbb37vxbqnq9887hmnnwlxk9lrw-python3.13-grpcio-tools-1.76.0-dist, file:///nix/store/ry6zrgh9rqalh10zmf8srps9yxp70lrr-python3.13-types-protobuf-6.32.1.20251105-dist, file:///nix/store/2y2dmszmqa77bm1sqirblbbfqwhssa1k-python3.13-wcwidth-0.2.13-dist, file:///nix/store/ddawnpambyf4bkzh2miy2k4h84gcisf3-python3.13-appdirs-1.4.4-dist, file:///nix/store/698wj6hpyvry0rf4f2i3y0914xlkk40l-python3.13-python-dateutil-2.9.0.post0-dist, file:///nix/store/7nqdp02jpjsvpbgnc8drlvzxcsdv6rb3-python3.13-docopt-0.6.2-dist, file:///nix/store/4x56qkrv47j749yq7b1zdfjgqnamcs9y-python3.13-ruamel-yaml-0.18.16-dist, file:///nix/store/1ilv8xccz04zkhwmgbp8s8ypjyaar94q-python3.13-docutils-0.21.2-dist, file:///nix/store/2v15jryir1lzaj0kz598rxvn1lzvms89-python3.13-lockfile-0.12.2-dist, file:///nix/store/hvp7ilsrcz22blp6whqarmnmyrmfk353-python3.13-certifi-2025.07.14-dist, file:///nix/store/si7j8vnvnc4g2dsvcj91akmasb3c865q-python3.13-charset-normalizer-3.4.3-dist, file:///nix/store/h4sr6hmfwp2knivbcg63lyc6xm8dr6z3-python3.13-idna-3.11-dist, file:///nix/store/aqsz3h6k92wiqf1bym6mwsaqayz5xdl1-python3.13-urllib3-2.6.0-dist, file:///nix/store/2ilhaf84677xbcdxwx7z0rxkbq1z5iy1-python3.13-alabaster-1.0.0-dist, file:///nix/store/xlh7myg32y73i9a8wywqawbznf2a6rhz-python3.13-babel-2.17.0-dist, file:///nix/store/6lgkslqwh9s3792yhn5b7b26hsc3mqq1-python3.13-imagesize-1.4.1-dist, file:///nix/store/h9w4pw75iincwb1f1ni8m49kghn4fh2c-python3.13-roman-numerals-py-3.1.0-dist, file:///nix/store/c3idj57rfd5lvdm8vz1dyicbwc33ibka-python3.13-snowballstemmer-3.0.1-dist, file:///nix/store/q5ya01g21b3fb9lpkvqmc8prgdn43scl-python3.13-sphinxcontrib-applehelp-2.0.0-dist, file:///nix/store/rqafar1ji5w5ay9idh4ckm6bvzr0bdjf-python3.13-sphinxcontrib-devhelp-2.0.0-dist, file:///nix/store/p8l0dipcmn63d9hmqpvvnkkvj3b8iks2-python3.13-sphinxcontrib-htmlhelp-2.1.0-dist, file:///nix/store/9lhgylzwarg08qq24qq1hwhcy9w9dkkm-python3.13-sphinxcontrib-jsmath-1.0.1-dist, file:///nix/store/l3xww66mssyjnwyqmjc4bqa189hsmzax-python3.13-sphinxcontrib-qthelp-2.0.0-dist, file:///nix/store/z1ng16h1xpyi6vzmyv0pva02c1nqba0n-python3.13-sphinxcontrib-serializinghtml-2.0.0-dist, file:///nix/store/fngs9v9djdx77fj7xfgcp8rd6chcc1qj-python3.13-sphinxcontrib-websupport-2.0.0-dist, file:///nix/store/j4y7lw169qqiif7nrcc8ypjc0vr8s792-python3.13-types-urllib3-1.26.25.14-dist, file:///nix/store/8bbagnvdkn0rb6nfmj8pvy2aw90dnfs0-python3.13-pycares-4.9.0-dist, file:///nix/store/5frkdvx76rbkh2nl6l8zqly5bmpim367-python3.13-pycparser-2.23-dist, file:///nix/store/pfadcqd2biy14vlx2fcn1rqh886z21pi-python3.13-parso-0.8.5-dist, file:///nix/store/6h9z1lyjhx9wqnmw8kym4rs04f33cy4n-python3.13-ptyprocess-0.7.0-dist, file:///nix/store/j86wjj7i3158cv81c9gv0y4ffkwh6bd0-python3.13-asttokens-3.0.0-unstable-2025-11-08-dist, file:///nix/store/knawlx9n06r13n2n4g60s6pin6bx4fb9-python3.13-executing-2.2.1-dist, file:///nix/store/dw2d5di6x9df0vmqk001fpj8f790afdi-python3.13-pure-eval-0.2.3-dist, file:///nix/store/fbbhi6f0hcykhngwlaf84xnmr6ygrdvk-python3.13-ply-3.11-dist, file:///nix/store/pn0ay7lclz09qqhg79y7nw5hnn5jp2dl-python3.13-flit-core-3.12.0-dist, file:///nix/store/zbi1j2viwbyvayjxj4bl4w1mfwan9hb2-python3.13-grpcio-1.76.0-dist, file:///nix/store/pxi68apljvaqkysq3fp48nyz9p8i1n4m-python3.13-setuptools-80.9.0-dist, file:///nix/store/lpwb2jg1dpyah1sc02l01achaf1fk560-python3.13-types-futures-3.3.8-dist, file:///nix/store/9wg2jxfjlhi63xkia81557fz0kdfs674-python3.13-ruamel-base-1.0.0-dist, file:///nix/store/2wydanqvnp8c2x8vzk2fzbxb61n90gdz-python3.13-ruamel-yaml-clib-0.2.14-dist, file:///nix/store/ng54hxs5awb8y5aw4y8yvqx3lks4vqzk-python3.13-pystemmer-2.2.0.1-dist, file:///nix/store/0x8qk0bx7rn484272pi155za1ia83bmj-python3.13-toml-0.10.2-dist
Requirement already satisfied: build in /nix/store/rjs7qfglhi7cgd3sbvgqhl6vajps3724-python3.13-build-1.3.0/lib/python3.13/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 2)) (1.3.0)
Requirement already satisfied: click in /nix/store/j7qlfl0lw3n1ykv5qa7rz6c1mxiyplcs-python3.13-click-8.3.1/lib/python3.13/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 3)) (8.3.1)
Requirement already satisfied: packaging in /nix/store/gx4vpfwx6djk15h6ki9raxlxkc7c1rba-python3.13-packaging-25.0/lib/python3.13/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 4)) (25.0)
Requirement already satisfied: pip-tools in /nix/store/8xkxgra8lckp2ismg2mdd1zz50911g0i-python3.13-pip-tools-7.5.2/lib/python3.13/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 5)) (7.5.2)
Requirement already satisfied: pip in /nix/store/59w24ywxs5lny5n9hx2zk1g29yy49iq4-python3.13-pip-25.0.1/lib/python3.13/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 6)) (25.0.1)
Requirement already satisfied: pyproject-hooks in /nix/store/rrbrry1vjvvf9vsqghq578s1bm5fy1sn-python3.13-pyproject-hooks-1.2.0/lib/python3.13/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 7)) (1.2.0)
Requirement already satisfied: setuptools in /nix/store/bq3172qx0c68yv0i4wy51y1bzln4wndr-python3.13-setuptools-80.9.0/lib/python3.13/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 8)) (80.9.0.post0)
ERROR: Could not find a version that satisfies the requirement tomli (from versions: none)
ERROR: No matching distribution found for tomli
[476/561] c++ obj/third_party/jsoncpp/repo/src/lib_json/jsoncpp.json_writer.cpp.o
[477/561] ACTION //src/controller/data_model:data_model_zapgen_generate(//build/toolchain/custom:custom)
[478/561] c++ obj/third_party/jsoncpp/repo/src/lib_json/jsoncpp.json_reader.cpp.o
[479/561] c++ obj/third_party/jsoncpp/repo/src/lib_json/jsoncpp.json_value.cpp.o
[480/561] c++ obj/src/controller/libChipController.CHIPDeviceController.cpp.o
[481/561] c++ obj/src/tracing/perfetto/perfetto.perfetto_tracing.cpp.o
[482/561] c++ obj/third_party/perfetto/repo/sdk/sdk.perfetto.cc.o
ninja: build stopped: subcommand failed.</pre>
</details>
<details>
<summary>python314Packages.home-assistant-chip-wheels</summary>
<pre>FAILED: [code=1] python/gen/matter_build_venv._install_base_3p_deps/pip_install_log.txt python/gen/matter_build_venv._install_base_3p_deps.pw_pystamp 
python3 ../../third_party/pigweed/repo/pw_build/py/pw_build/python_runner.py --gn-root ../../ --current-path ../.. --default-toolchain=//build/toolchain/custom:custom --current-toolchain=//third_party/pigweed/repo/pw_build/python_toolchain:python --touch python/gen/matter_build_venv._install_base_3p_deps.pw_pystamp --capture-output --module pip --python-virtualenv-config python/gen/matter_build_venv/venv_metadata.json --python-dep-list-files python/gen/matter_build_venv._install_base_3p_deps_metadata_path_list.txt -- install --requirement ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt --log python/gen/matter_build_venv._install_base_3p_deps/pip_install_log.txt
Looking in links: file:///nix/store/y6hd36lr59qrd0x1s0rb61i1012k2yjw-python3.14-aiohttp-3.13.2-dist, file:///nix/store/gha99al87a30c933zmj5s40nm5p7iy88-python3.14-alive-progress-3.1.5-dist, file:///nix/store/s1781j11rpri9q7b7v6i0q4pk0g3njdm-python3.14-colorama-0.4.6-dist, file:///nix/store/pdqm6jr4n749yi7hkn8xjh2b1iqcibp2-python3.14-coloredlogs-15.0.1-dist, file:///nix/store/z805r6b86ykymkhk1kvjsngadi9pyam6-python3.14-click-8.3.1-dist, file:///nix/store/bpngx8d1ixz6jpa21hm0dxaz69q9kssl-python3.14-cryptography-46.0.3-dist, file:///nix/store/a2ahf6wbbrhpsjwbr2mh1rmqlx21dgm1-python3.14-debugpy-1.8.19-dist, file:///nix/store/paw62g0zibx9405q5p89gbpbg9qswb37-python3.14-diskcache-5.6.3-dist, file:///nix/store/ihd1y0qpm6wrd1xkaxhmqfy0xf83bxqy-python3.14-googleapis-common-protos-3.31.3-dist, file:///nix/store/gzbgby3zjb9l47ibyns6515dpj6y705s-python3.14-ipython-9.7.0-dist, file:///nix/store/bs4qmawp46a8awb72fg460ifsaig9cpl-python3.14-jinja2-3.1.6-dist, file:///nix/store/4dd9wkyyzqx694lyknfcfs2nd4pzm8iq-python3.14-json5-0.12.1-dist, file:///nix/store/92lgnf2bkdxp5px3kdxc0wsb4z2yz2dp-python3.14-jsonschema-4.25.0-dist, file:///nix/store/sm1dhp3x72k0dh6ickpg322g3i4w9lkv-python3.14-lark-1.2.2-dist, file:///nix/store/c4yqj2wvfyi3pg0bhfwznw5v8qmypwid-python3.14-mobly-1.13-dist, file:///nix/store/dvm5zcq3ldxd83jha80wzi3mhm928yba-python3.14-mypy-1.17.1-dist, file:///nix/store/qhivd41wg39y5d334ldlylwhlvm3g5zg-python3.14-mypy-protobuf-3.6.0-dist, file:///nix/store/3ggj6niz6z6izvmxh30ifixk54wflhl4-python3.14-packaging-25.0-dist, file:///nix/store/rd2liwqa6zs0930hk0722p82qi78fbxl-python3.14-parameterized-0.9.0-dist, file:///nix/store/ppq4cfrdkw0j91cipvyqczn8w4s7bbzg-python3.14-pkgconfig-1.5.5-dist, file:///nix/store/habfy3rc9m39j7a7q389ph958c3bbl0s-python3.14-prompt-toolkit-3.0.52-dist, file:///nix/store/l272n1cg1yw5kqkjff3biy26cwv472yr-python3.14-protobuf-6.33.2-dist, file:///nix/store/w954lwi6p7f24gakmg1cl77xaqj4292a-python3.14-psutil-7.1.3-dist, file:///nix/store/kvcka1fwrgzrhh98hixv9rv0xdvs65z0-python3.14-ptpython-3.0.32-dist, file:///nix/store/n47nz6b0jcg2pvixl89xqkc9j8n52600-python3.14-pyfakefs-5.9.2-dist, file:///nix/store/gvx2sq9hdqfl58c55kcr8dfnzsh5d4sw-python3.14-pyelftools-0.32-dist, file:///nix/store/8s8c3hpr2zarkiiggv03423x2rlq00k3-python3.14-pygments-2.19.2-dist, file:///nix/store/d42700b7gxbiwhp4ljc0a5dha4bn9cgz-python3.14-pykwalify-1.8.0-dist, file:///nix/store/390hmbnnari4r84pyzs38w6innni8x5d-python3.14-pyperclip-1.11.0-dist, file:///nix/store/zmv955gqmbwhrqkn3rqx7ll1h8ysadyy-python3.14-pyserial-3.5-dist, file:///nix/store/mx5xwhh3jb6idy6i0xj6x5nvdgin4i7w-python3.14-python-daemon-3.1.2-dist, file:///nix/store/qab6xy7vy0pzc6mkmwj4qnv5p4xihfy1-python3.14-python-path-0.1.3-dist, file:///nix/store/jsliq1mvx9wx02i52vyvr2c2h79w2zlz-python3.14-pyyaml-6.0.3-dist, file:///nix/store/sp6jr97j3rhk5n6gmx5s28jv21zaf6rl-python3.14-requests-2.32.5-dist, file:///nix/store/s3fw3h21klnw6r0dm7mj78wrhg5m0d85-python3.14-six-1.17.0-dist, file:///nix/store/s0npybpv3pybvv733izx7a1jk3qfq43z-python3.14-sphinx-8.2.3-dist, file:///nix/store/0rzdmicrm2ynv0krz8wsk4x31wbm65b0-python3.14-sphinx-argparse-0.5.2-dist, file:///nix/store/91044i7p5s623jgm5b58lb0xsgqnk0aq-python3.14-sphinx-design-0.6.1-dist, file:///nix/store/9zjivyq6bpqhk1j97h8z5jcm5q89g6lv-python3.14-tabulate-0.9.0-dist, file:///nix/store/sfh0vv3hy3wy121sr3iv49zzzi53x79a-python3.14-tornado-6.5.2-dist, file:///nix/store/j6vyf4vvdss5ir6jxaxdsvapxfjd07ak-python3.14-types-pyyaml-6.0.12.20250516-dist, file:///nix/store/qamvhpcj3q48rwfwdb2bcw20c65xg2r6-python3.14-types-requests-2.32.4.20250913-dist, file:///nix/store/pscxfyxhzpn25w1v5f8k83jqqg13kfyq-python3.14-watchdog-6.0.0-dist, file:///nix/store/n8gfp4d9qk5ssh926z0lpyahsfr4qck7-python3.14-websockets-15.0.1-dist, file:///nix/store/bc5bv4vga3w770zi6vcgsd3ywvj0khjs-python3.14-aiohappyeyeballs-2.6.1-dist, file:///nix/store/20cvx9nvzlgsh1qqgq1hda952phv3rjj-python3.14-aiosignal-1.4.0-dist, file:///nix/store/b88lpksp7c8fz9ff3wwwqf3wk80zqyan-python3.14-attrs-25.4.0-dist, file:///nix/store/wy74kw93hm26bk3kwyya59m7fmwgvm2s-python3.14-frozenlist-1.7.0-dist, file:///nix/store/b2rcxw5kfxb7hxp72m15x67zkgr30ci0-python3.14-multidict-6.7.0-dist, file:///nix/store/zj78krwixbagr4233x7zlaixm69ahvw4-python3.14-propcache-0.4.1-dist, file:///nix/store/ymccjn52979qjrfsqap5hvqszc1ci17g-python3.14-yarl-1.22.0-dist, file:///nix/store/x0y6jfs41q48r2j1nzj1zjjakhv0c88d-python3.14-aiodns-3.5.0-dist, file:///nix/store/8hmijhhy6x0cj30zpk24sr86ch9v4n8x-python3.14-brotli-1.2.0-dist, file:///nix/store/b2n62vhr4i8cf44yhrv8hbq0y1im0n9s-python3.14-about-time-4.2.2-dist, file:///nix/store/gliqc130fyvbr7v11qn09y869z2b1s20-python3.14-grapheme-0.6.0-dist, file:///nix/store/jfvnl9prq4lp9gh99wahkd23fwk28qsm-python3.14-humanfriendly-10.0-dist, file:///nix/store/y9k9sdwljsqlf05vbqf4360gymahw6jf-python3.14-cffi-2.0.0-dist, file:///nix/store/hj43hqikcmviidrcimpls6mz3q8h0am0-python3.14-decorator-5.2.1-dist, file:///nix/store/hqmqkppfk581gqmj15w8sd5ww4s9zlmg-python3.14-ipython-pygments-lexers-1.1.1-dist, file:///nix/store/q2qzkj67rcwz2cn713a1956gj6z8c64n-python3.14-jedi-0.19.2-dist, file:///nix/store/q3zcmxcxsdhc5xl5h01v9n7mvkw975p7-python3.14-matplotlib-inline-0.1.7-dist, file:///nix/store/kgq50sf51kfvhn1is2xkn2lsniyk5rg0-python3.14-pexpect-4.9.0-dist, file:///nix/store/3aasiy8z8n8mwb6w2yfr8hay0b09ccbl-python3.14-stack-data-0.6.3-dist, file:///nix/store/9aza6xy7d306kkgqazn9fwlw7s1pmgwk-python3.14-traitlets-5.14.3-dist, file:///nix/store/nkf2r2a8j4b3mhz1hc1ab62z42dgk73q-python3.14-markupsafe-3.0.3-dist, file:///nix/store/1cir09i8362lx3xm2im4xs34zh0wjyfr-python3.14-jsonpath-ng-1.7.0-dist, file:///nix/store/d6696a6qiagx3cfwzmv52zf973lapx0i-python3.14-jsonschema-specifications-2025.4.1-dist, file:///nix/store/rvf60akinl79z5cm6k52r51r2a71llw3-python3.14-referencing-0.36.2-dist, file:///nix/store/8xbl015fqzj9g6x7fnwbvqvv74hdgvn5-python3.14-rpds-py-0.28.0-dist, file:///nix/store/qnimi0dpcb65blgdv1rr41dfsmpfgh9l-python3.14-regex-2025.9.18-dist, file:///nix/store/mwdz1j2q8hd7ihfimd28r4hsz28xi5ks-python3.14-portpicker-1.6.0-dist, file:///nix/store/27fxkbjpmcjl9bkb333gyqw7whvazaw4-python3.14-timeout-decorator-0.5.0-dist, file:///nix/store/arq0wchjvqs7yzkscygafzw93k1cqawx-python3.14-typing-extensions-4.15.0-dist, file:///nix/store/cisjkjkiva6838gzkn86q4knqkhas1mx-python3.14-mypy-extensions-1.1.0-dist, file:///nix/store/2rw7cz9nymn97d6bqcn5j201g88bwnhw-python3.14-pathspec-0.12.1-dist, file:///nix/store/kyrdl7zmpivg3l1s5n4nq4wbvrg3dgz7-python3.14-grpcio-tools-1.76.0-dist, file:///nix/store/xj7x46b2jpdjsm36sfl0z4dmg0wy2glc-python3.14-types-protobuf-6.32.1.20251105-dist, file:///nix/store/1hcd612psig61fxia1h2kvwvl2wg8svq-python3.14-wcwidth-0.2.13-dist, file:///nix/store/jfwqhbc6w0dbvicbqcxska7c5z94fd09-python3.14-appdirs-1.4.4-dist, file:///nix/store/f9pxdch89n2akjccg1svi327wi0msckc-python3.14-python-dateutil-2.9.0.post0-dist, file:///nix/store/5c3d0c8sbsh25187zipznh2l1pahiik1-python3.14-docopt-0.6.2-dist, file:///nix/store/5m4339ssn7j6vrs3ghm290ybchcjlgsk-python3.14-ruamel-yaml-0.18.16-dist, file:///nix/store/z8kij5f3dpisr6zrv0pjrsrala1wqkpf-python3.14-docutils-0.21.2-dist, file:///nix/store/14rri9fjvybiz2gbg1l1h1d54w7a54jp-python3.14-lockfile-0.12.2-dist, file:///nix/store/q31s4h90g6xm7p0xb4mpnyhzad60z2c4-python3.14-certifi-2025.07.14-dist, file:///nix/store/p4f0i848659kin34bax2qsvgsj4pg4nv-python3.14-charset-normalizer-3.4.3-dist, file:///nix/store/75w88cllf2b26x3z8k8hm8zi10kq26w7-python3.14-idna-3.11-dist, file:///nix/store/6xg5aaxc2nk5pxcgzsqgfdsrvxd7dg1m-python3.14-urllib3-2.6.0-dist, file:///nix/store/gqp37fcqpl96n4w0bkd0mqrf9xxdklwm-python3.14-alabaster-1.0.0-dist, file:///nix/store/84ffgasvwl6bcwdgvszi082cvqq1ck8f-python3.14-babel-2.17.0-dist, file:///nix/store/ymcm8v4qxxr42sp6y70zsi56dy9ibx41-python3.14-imagesize-1.4.1-dist, file:///nix/store/5skh6lzapx3xpqiz8q9jfq97ig1dhkf1-python3.14-roman-numerals-py-3.1.0-dist, file:///nix/store/6v2yh6v7jsrmm99dvci57a2xs72gf2hx-python3.14-snowballstemmer-3.0.1-dist, file:///nix/store/vapkn8y6shhp7441x3x56vsd9d6zi2qb-python3.14-sphinxcontrib-applehelp-2.0.0-dist, file:///nix/store/zilw2jvv7y01j1l6pm0fgrf3dlml0qd6-python3.14-sphinxcontrib-devhelp-2.0.0-dist, file:///nix/store/pqczxwfn0hcdrzs8q3c8063a518bv971-python3.14-sphinxcontrib-htmlhelp-2.1.0-dist, file:///nix/store/j6xgzbjzlfsbzmf039v631jx90kpnknb-python3.14-sphinxcontrib-jsmath-1.0.1-dist, file:///nix/store/n576f6v8z9sfhfsaghn69hqasi2ydga2-python3.14-sphinxcontrib-qthelp-2.0.0-dist, file:///nix/store/dcb7s07p266vql8k4vymx2mbbqnbmg3p-python3.14-sphinxcontrib-serializinghtml-2.0.0-dist, file:///nix/store/x82qflz791483k319ffzx1hys69id0x2-python3.14-sphinxcontrib-websupport-2.0.0-dist, file:///nix/store/ll90n59v2avl2i9dy5wigmdyd3nhfvw3-python3.14-types-urllib3-1.26.25.14-dist, file:///nix/store/nl8imv7sb5v6pj880v0bwg8mick412ij-python3.14-pycares-4.9.0-dist, file:///nix/store/yqysm96dqmgm7iskfyvp6x0yfgm80xc6-python3.14-pycparser-2.23-dist, file:///nix/store/swjsax83anbd348502wxx1h1b1r69dgs-python3.14-parso-0.8.5-dist, file:///nix/store/6lqyqnvnz2a3l49wgla5rdwnbgwlnf7s-python3.14-ptyprocess-0.7.0-dist, file:///nix/store/vpkhdzd325k71b8s3q6y49i0rdihpl09-python3.14-asttokens-3.0.0-unstable-2025-11-08-dist, file:///nix/store/i9jrmwmzsd09lkla1asf5nmjsrfpwijh-python3.14-executing-2.2.1-dist, file:///nix/store/ixxj8b295mis4ccfsk4dl4gmdm7k5g02-python3.14-pure-eval-0.2.3-dist, file:///nix/store/15vmbbljriydh8xb6127c99c8ap37kvj-python3.14-ply-3.11-dist, file:///nix/store/51impz1ck3yknh4xjqk6381hwfcfh1a8-python3.14-flit-core-3.12.0-dist, file:///nix/store/8vflmvhhq75kwbsmydgzqyisab3wagck-python3.14-grpcio-1.76.0-dist, file:///nix/store/md7h90akqbhqvw2hnz44sq2f9hkj3jry-python3.14-setuptools-80.9.0-dist, file:///nix/store/kbkrkncp6l30wpfx1vayp7yn0lji2m7g-python3.14-types-futures-3.3.8-dist, file:///nix/store/g4gllswbris7nrny4gp2g89fy6sqsn6q-python3.14-ruamel-base-1.0.0-dist, file:///nix/store/369ksl01if5i5nra3jffr3qwlhbm2qd9-python3.14-ruamel-yaml-clib-0.2.14-dist, file:///nix/store/g1r123239fdlikjm2dyglb20fcdnpfjr-python3.14-pystemmer-2.2.0.1-dist, file:///nix/store/vlxdaql6sgmvclgwp6ixvab32vki76qb-python3.14-toml-0.10.2-dist
Requirement already satisfied: build in /nix/store/pj4k2nqjc3q98r00wqm5xchjkzvx2fnp-python3.14-build-1.3.0/lib/python3.14/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 2)) (1.3.0)
Requirement already satisfied: click in /nix/store/17xz27im84vgpbibxpazd5wjf0y29w6r-python3.14-click-8.3.1/lib/python3.14/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 3)) (8.3.1)
Requirement already satisfied: packaging in /nix/store/b0z5vgmamp4p79hr1fl830hhjb6myvj0-python3.14-packaging-25.0/lib/python3.14/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 4)) (25.0)
Requirement already satisfied: pip-tools in /nix/store/crjdwah81jndpb80c5y3nkd3hdd1mifq-python3.14-pip-tools-7.5.2/lib/python3.14/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 5)) (7.5.2)
Requirement already satisfied: pip in /nix/store/zlap6vnwdpxmrikn15qhlz81rjysdcam-python3.14-pip-25.0.1/lib/python3.14/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 6)) (25.0.1)
Requirement already satisfied: pyproject-hooks in /nix/store/f75plbpx9zslh1vdknqx6b24k5m3lg86-python3.14-pyproject-hooks-1.2.0/lib/python3.14/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 7)) (1.2.0)
Requirement already satisfied: setuptools in /nix/store/nka950z7rk9smv94ywkvswx1nxwh1n7r-python3.14-setuptools-80.9.0/lib/python3.14/site-packages (from -r ../../third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/virtualenv_setup/python_base_requirements.txt (line 8)) (80.9.0.post0)
ERROR: Could not find a version that satisfies the requirement tomli (from versions: none)
ERROR: No matching distribution found for tomli
[477/561] ACTION //:matter_build_venv._generate_3p_requirements(//third_party/pigweed/repo/pw_build/python_toolchain:python)
[478/561] c++ obj/third_party/jsoncpp/repo/src/lib_json/jsoncpp.json_value.cpp.o
[479/561] c++ obj/third_party/jsoncpp/repo/src/lib_json/jsoncpp.json_writer.cpp.o
[480/561] c++ obj/third_party/jsoncpp/repo/src/lib_json/jsoncpp.json_reader.cpp.o
[481/561] c++ obj/src/controller/libChipController.CHIPDeviceController.cpp.o
[482/561] c++ obj/src/tracing/perfetto/perfetto.perfetto_tracing.cpp.o
[483/561] c++ obj/third_party/perfetto/repo/sdk/sdk.perfetto.cc.o
ninja: build stopped: subcommand failed.</pre>
</details>
</details>

--- a/pkgs/development/python-modules/home-assistant-chip-wheels/default.nix
+++ b/pkgs/development/python-modules/home-assistant-chip-wheels/default.nix
@@ -54,6 +54,7 @@
   sphinx-design,
   stdenv,
   tabulate,
+  tomli,
   tornado,
   types-pyyaml,
   types-requests,
@@ -232,6 +233,7 @@ stdenv.mkDerivation rec {
         sphinx-argparse
         sphinx-design
         tabulate
+        tomli
         tornado
         types-pyyaml
         types-requests

